src/kboot.c

```diff
@@ -396,6 +397,20 @@ static int dt_set_memory(void)
         }
     }
 
+    if (chip_id == T8012) {
+        u64 cache_min = ALIGN_UP(dram_max, BIT(29));
+        u64 cache_max = dram_base + dram_size;
+
+        /* Set x86 SSD Cache as OS memory, any potential driver would need the memory described
```

Comment:
This won't work with an actual driver, it just abuses this region as main memory. If a driver were to use this, it would have to be described as reserved-memory (which automatically adds it to the list if mapped).

If you intend to ever actually support ANS here, then this probably needs to be kept reserved.

Reply:
I think I will not describe this memory region for now, given that it might need new compatibles and stuff and that that part of the hardware is not well understood. In this case, this memory region only needs to be concerned while correcting `mem_size_actual`.